content/docs/08-troubleshooting/01-migration-guide/04-migration-guide-4-0.mdx

```diff
@@ -62,3 +62,9 @@ Please use the `headers` and `body` parameters instead.
 ### Removed `name` from UI message
 
 The `name` property from the `Message` type has been removed.
+
+### Increased default size of generated IDs
+
+The [`generateId`](/docs/reference/stream-helpers/generate-id) function, as well
+as generators created with `createIdGenerator`, now generate 16-character IDs.
```

Comment:
We have not exposed `createIdGenerator` yet in `ai`. Let's only mention `generateId` here. The `generateId` reference docs might also need to be updated.

Reply:
`createIdGenerator` looks like it is (and has been) exported for a bit at least, but it isn't mentioned in any docs. Updated docs here to remove mention.

Updated `50-generate-id.mdx` to note the new default as well.